params.yml

```diff
@@ -23,6 +23,9 @@ deploy_zkevm_permissionless_node: true
 # Deploy observability stack.
 deploy_observability: true
 
+# Deploy blutgang.
```

Comment:
What do you think of naming this `deploy_eth_loadbalancer` or something like that? I think that's a little bit more generic (e.g. in the case we want to migrate to another solution that is not blutgang).